serverlessworkflow/modules/ROOT/pages/eventing/consume-producing-events-with-kafka.adoc

```diff
@@ -88,11 +87,16 @@ mp.messaging.incoming.wait.value.deserializer=org.apache.kafka.common.serializat
 mp.messaging.outgoing.resume.connector=smallrye-kafka
 mp.messaging.outgoing.resume.value.serializer=org.apache.kafka.common.serialization.StringSerializer
 ----
+--
+
+Default channel mapping::
++
+--
+The link:{events_example_url}[`serverless-workflow-events-quarkus`] example application uses two CloudEvent types, including `applicants` (incoming) and `decisions` (outgoing).
 
-==== Using default channel
+As mentioned before, Kafka topic names matches the CloudEvent types. However, to use the default channel functionality, `kogito_incoming_stream` and `kogito_outgoing_stream` are used as channel names. If there is no specific channel name for the CloudEvent type, then the default channels are used. Also, the default channels need to be mapped to the desired topic name using the `topic` property.
```

Comment:
As mentioned before should be as in the previous case. 
Im trying to explain the reader that we are using default channel mapping here just for explanation purposes

Reply:
Done